### PR TITLE
Add a skeleton level of Main/ Notebook UI

### PR DESCRIPTION
**Pull request by AhyoungRyu**
#### Main page
Screen Shot 2019-11-26 at 10 53 07 AM

- [x] Global navigation (only includes a logo & search bar UI components)
- [x] Listing notebook (but don't  support the folder structure yet)

#### Notebook page
Screen Shot 2019-11-26 at 10 52 54 AM

- [x] Paragraph listing 
- [x] Added `react-ace` for the paragraph editor component and display the paragraph editor text (didn't do for the result text 😅). I wanted to try Monaco editor in this time but failed to bring in due to CSS conflict issue. https://github.com/react-monaco-editor/react-monaco-editor#using-with-webpack <- this didn't solve my issue
- [x] Notebook navbar with the action toolbar

### CODE REVIEW

File: images/icons/zeppelin_svg_logo.svg [hunk 1]
```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (sravan-s) [inline images/icons/zeppelin_svg_logo.svg:1]: Is it okay to use this asset?

File: webpack.config.dev.js [hunk 2]
```diff
@@ -10,8 +11,12 @@ export default {
     // To support react-hot-loader
     alias: {
       '~': path.resolve(__dirname, './src'),
+      '~images': path.resolve(__dirname, './images'),
       'react-dom': '@hot-loader/react-dom',
     },
+    plugins: [
```
Comment (sravan-s) [inline webpack.config.dev.js:17]: Any specific reason for using this plugin?

Comment (AhyoungRyu) [reply webpack.config.dev.js:17]: It's for `import Path from "~/Some/Dir/Path/ndex.js"` -> `import Path from "~/Some/Dir/Path"`.  Or is there any workaround to make this work without using this plugin?

Comment (sravan-s) [reply webpack.config.dev.js:17]: I mean I didnt see such usages 👀

**Review by sravan-s — APPROVED** [review]
LGTM

We should make the lint more strict :)

File: webpack.config.dev.js (hunk 2 above)
Comment (AhyoungRyu) [reply webpack.config.dev.js:17]: It's for this part 
https://github.com/gogumaa/zeppelin-web/pull/12/files#diff-f880b57906a072cdffa62da2c954ddd4R12

Comment (sravan-s) [reply webpack.config.dev.js:17]: ah, cool, the weird thing is that I have never noticed this thing in my projects :)

File: images/icons/zeppelin_svg_logo.svg (hunk 1 above)
Comment (AhyoungRyu) [reply images/icons/zeppelin_svg_logo.svg:1]: Yeah I think so? 
I downloaded from the official website https://zeppelin.apache.org/assets.html.

File: webpack.config.dev.js (hunk 2 above)
Comment (AhyoungRyu) [reply webpack.config.dev.js:17]: Right I know what you mean. I actually was about to ask you lol 
I also couldn't find where we make this possible in Zepl webapp  webpack configuration 😅

**Pull request merged**